src/bidiMapper/domains/protocol/parsers/browsingContextParser.ts

```diff
@@ -0,0 +1,41 @@
+import { InvalidArgumentErrorResponse } from '../error';
+import { BrowsingContext } from '../bidiProtocolTypes';
+
+export namespace BrowsingContextParser {
+  export class BrowsingContextParser {
+    public static parseOptionalList(
+      contexts: any
+    ): BrowsingContext.BrowsingContext[] | undefined {
+      if (contexts === undefined) return undefined;
+      if (!Array.isArray(contexts))
+        throw new InvalidArgumentErrorResponse(
+          "Wrong format parameter 'contexts'. Not an array."
+        );
+
+      // Get all the bad values.
+      const wrongContexts = contexts.filter((c) => {
+        // Return `true` if param is not correct.
+        if (!c) return true;
+        if (!(c instanceof String)) return true;
```

Comment:
I think you meant typeof c === 'string' ?

Reply:
done in helper